### ISSUE TITLE
Epic #24 - Application Start/End pages

### ISSUE THREAD

**Issue by nmadruga** [openmsupply/application-manager-web-app#231]
## Description
Add the start and end messages to each template and display as part of the application process, the Applicant should see the start page before starting a new application and the end page after the submission. The end page will be included in the summary page after the application was submited.

### Sprint 14:
Issue 1 (Effort 1?)
- Add two new TEXT fields to `template` table: `start_message`, `submission_message`
   - Submission message should have a default: "Thank you... etc.", but NOT the start message
- Write start and submission messages for existing templates (put in `insertData.js`)

## Issue 2 (Effort 2?)
- Front-end update to query these fields in the getTemplate query
- Display start message in ApplicationStart (ideally using evaluator, though that could wait until we create [stringSubstitution operator](https://github.com/openmsupply/application-manager-server/issues/158))
- If start_message is NULL, automatically create the application and go to the first page (we'd use this for userRegistration)

## Issue 3 (Effort 2-3?)
- Show submission message on Summary page upon successful submission (this is what we agreed we'd do, right? Not a new page?)
- Hide Submit button and Edit buttons after successful submission
- Show alert (modal?) if submission unsuccessful before re-directing to first invalid page (this part already done)

**Comment by CarlosNZ** [comment]
Can we add Markdown integration into this epic (new issue), since it seems pretty fundamental to displaying the start and submit messages properly?

(Also bearing in mind that markdown interpretation will be required elsewhere such as in application fields, so we need to pick a suitable markdown library that can be used throughout the application)
